### ISSUE TITLE
Color the mean shapes in a panel 

### ISSUE THREAD

**Issue by raz1** [MomX/Momocs#240]
Hi,

I wish to color the mean shapes in a panel (different colors) using this code:
```
ms <- MSHAPES(data_f, 'type')
Out(ms$shp) %>% panel(names=T, fac='type',palette = col_spring)

```
I get this error:
`not a valid column specification, returning NULL`

Any idea why and how to solve it?

**Comment by vbonhomme** [comment]
Yep, type (and fac) no longer exist in the freshly created Out !

**Comment by raz1** [comment]
Can it be solved? it will be just visualize nice to color each mean shape

**Comment by vbonhomme** [comment]
Yes, simply create a tibble with, say, enframe using the names !
